### ISSUE TITLE
voice over does not announce who the message is from in a chat

### ISSUE THREAD

**Issue by lilmike** [signalapp/Signal-iOS#1211]
- [x] I have searched open and closed issues for duplicates

---
### Bug description

Describe here the issue that you are experiencing.

When using voice over, the messages do not say who they are from. So, where sighted people can tell by a color whether the message is from or to them, voice over users cannot.
### Steps to reproduce
- using hyphens as bullet points
- list the steps
- that reproduce the bug
1. Turn on voice over from settings, or with a triple tap of the home button.
2. Enter signal, and go to a chat where you have both sent and received messages.
3. Notice that when placing your finger on the message or flicking through them, voice over simply reads the message, and does not say whether it is from or to you, and if to you, who from.

**Actual result:** Describe here what happens after you run the steps above (i.e. the buggy behaviour)
**Expected result:** Describe here what should happen after you run the steps above (i.e. what would be the correct behaviour)
### Device info

**Device**: iPhone 5s
**iOS Version**: iOS 9.2

**Comment by cannona** [comment]
I am also a VO user, and so am also effected by this issue.  If there is anything I can do to help get this resolved, please don't hesitate to ask.

TL;DR: me too!
